tests/integration/targets/vm_snapshot/tasks/01_vm_snapshot_info.yml

```diff
@@ -0,0 +1,172 @@
+---
+- name: Test vm_snapshot_info
+  vars:
+    nonexistent_snapshot_label: no-label
+    nonexistent_snapshot_vm_name: no-vm-name
+    nonexistent_snapshot_serial: -1
+    non_unique_snapshot_label: not-unique
+    created_snapshot_test_vm_1:
+      vm_name: snapshot-test-vm-1
+      snap_labels:
+        - snap-0
+        - snap-1
+        - snap-2
+    created_snapshot_test_vm_2:
+      vm_name: snapshot-test-vm-2
+      snap_labels:
```

Comment:
we need snap-0, snap-1, snap-2 also here. We need to differentiate snapshot vm-1 snap-0 from vm-2 snap-0.